packages/perseus/src/mixins/provide-keypad.tsx

```diff
@@ -11,7 +11,7 @@
  * extend a `ProvideKeypad` component instead of using this mixin.
  */
 
-import {MobileKeypad} from "@khanacademy/math-input";
+import {KeypadContext, MobileKeypad} from "@khanacademy/math-input";
```

Comment:
No action needed: just for reference, both [ProvideKeypad](https://khanacademy.atlassian.net/browse/LC-386) and [ItemRenderer](https://khanacademy.atlassian.net/browse/LC-408) are dead code now. It's fine if it's out of scope for this PR, but I also wouldn't spend much more time updating them.